tokens/_options/space.json5

```diff
@@ -1,14 +1,17 @@
 {
   options: {
     space: {
-      'squish': {
-        value: '0.5',
-        category: 'size',
-      },
-      'stretch': {
-        value: '1.5',
-        category: 'size',
-      },
+      'squish': 0.5,
```

Comment:
We may want to organize this differently, but there are few enough values where it may not be worth it?